src/content/learning-resources-index/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Flutter learning resources
+description: A catalog of Flutter sample applications, codelabs, and tutorials.
+short-title: Learning resources
+show_breadcrumbs: false
+layout: wide
+toc: false
+js: [ { url: '/assets/js/learning-resources-index.js', defer: true } ]
+---
+
+:::secondary
+This page lists all of our additional learning resources:
+* Cookbook recipes that demonstrate how to solve common problems with Flutter.
+* Guided Codelabs and tutorials that walk you through building features and applications.
+* Working sample applications that show how to use Flutter.
+* Feature-rich demo applications that show how larger applications are built.
+:::
+
+{% assign resources = learning-resources-index.codelabs | concat: learning-resources-index.cookbook | concat: learning-resources-index.demos | concat: learning-resources-index.quickstarts_flutter | concat: learning-resources-index.quickstarts_dart | shuffle -%}
+
+{% assign filters = learning-resources-index.filters -%}
+
+<div id="resource-index-content">
+    <div class="left-col" id="resource-index-main-content">
+        <div 
+            id="resource-search-group" 
+            class="chip-filters-group" 
+            style="margin-bottom:20px"
+        >
+            <div class="top-row">
+                <div class="search-wrapper" id="resource-search">
+                    <span class="material-symbols leading-icon" aria-hidden="true">search</span>
+                    <input type="search" placeholder='Try "button" or "networking"...'
+                        aria-label="Search learning resources by name and category">
+                </div>
+                {% comment -%}This dropdown is shown on narrow screens{% endcomment -%}
+                <button class="show-filters-button">
+                    <span class="material-symbols" aria-hidden="true">filter_list</span>
+                </button>
+            </div>
+            <div class="label-row">
+                <label for="resource-search">
+                    Showing <span id="displayed-resource-card-count">0</span> / <span id="total-resource-card-count">0</span>
+                </label>
+                <button id="clear-resource-index-filters">
```

Comment:
Can you disable the button when no search query or filter is specified? Otherwise it's potentially confusing as clicking it results in no action.